### DISCUSSION TITLE
Add copy to clipboard in new Dashboard

### DISCUSSION THREAD

**Discussion by ChristianGK-GC** [2Abendsegler/GClh#2350] [Ideas]
Eine einfache copy Funktion des Log Textes im neuen Dashboard fände ich sehr cool.
Dann müsste man nicht immer umständlich über "View log" gehen um den Text kopieren zu können.

Das könnte dann so aussehen:
![grafik](https://user-images.githubusercontent.com/24353480/218784428-d376156c-9156-4877-bd49-ce3d44ba73c3.png)

**Comment by 2Abendsegler** [comment]
Hört sich gut an.

---
Off Topic: Deine Verpixelungen sehen viel netter aus als meine. Womit machst du sie?

**Reply by ChristianGK-GC** [reply]
Ich verwende [Greenshot](https://getgreenshot.org/downloads/)
Pixel size: 5 
![grafik](https://user-images.githubusercontent.com/24353480/218884914-dd6b16fa-75d5-4481-bfb5-f8f25d2f38d8.png)

**Reply by 2Abendsegler** [reply]
Super, danke!

**Comment by ChristianGK-GC** [comment]
@2Abendsegler Schau doch mal bitte drüber.
[Latest-Activity-Copy-to-clipboard-for-logtext](https://github.com/ChristianGK-GC/GClh/tree/Latest-Activity-Copy-to-clipboard-for-logtext)

**Comment by 2Abendsegler** [comment]
Sieht gut aus!
Musst aber trotzdem noch einen PR machen. 😀
Wo ich schon mal bei dir war habe ich dir auch mal ein Sternchen gegeben. Wenn du den Fork löschst dann ist das womöglich wieder weg. Aktualisieren des Forks sollte ja aber auch gehen.

**Reply by ChristianGK-GC** [reply]
Soll ich dafür noch ein Setting machen für die config?

**Reply by ChristianGK-GC** [reply]
Off Topic: Du bist aber früh auf gewesen...

**Reply by 2Abendsegler** [reply]
> Soll ich dafür noch ein Setting machen für die config?

Nein, nicht nötig.
